### ISSUE TITLE
Cannot obtain comparable performance

### ISSUE THREAD

**Issue by yushuinanrong** [titu1994/Image-Super-Resolution#8]
@titu1994
Hi,
First I want to thank you for your great work! It is very inspiring.
Right now, I'm trying the very basic CNN for SR ("BaseSuperResolutionModel") use your code. Basically, what I've done is calling the img_utils.py function (scale = 3) to extract patches and then calling the test.py function. 
I'm using the "set 5" dataset as validation set. After 100 epochs, the training PSNR is close to 28.3dB and validation PSNR is close to 27.4dB. The loss did not decrease after 100 epochs.
I left all the other parameters as default. 

I noticed you commented that you also got worse performance comparing to that reported in the paper. But 27.4 dB is way worse than the number in the paper (32.6dB).

I'm wondering if you witnessed similar performance as I reported.

Thanks

**Comment by titu1994** [comment]
I agree that the model training isn't working correctly.

It is most probably a problem with the patch generation, or patch noisiness or some other factor. Currently I don't have enough time to look for a fix, but if you are so inclined, please attempt a fix and add a pull request.

**Comment by yushuinanrong** [comment]
Thanks for your prompt response.

I actually implemented some more sophisticated models myself, but none shows promising performance. 
I agree it is likely due to the pre-processing step. I will keeping trying and let you know if I figure it out.

Thanks
